app/assets/javascripts/controllers/container_dashboard/util/dashboard-utils-factory.js

```diff
@@ -97,8 +97,28 @@ angular.module('miq.util').factory('dashboardUtilsFactory', function() {
   var parseDate = function(date) {
     myDate = Date.parse(date);
     return isNaN(myDate) ? date : myDate
-  }
+  };
+  var iconClassForProvider = function(provider) {
+    switch (provider) {
+      case "openshift":
+        iconClass = "pficon pficon-openshift";
+        break;
+      case "openshift_enterprise":
+        iconClass = "pficon pficon-openshift";
+        break;
+      case "kubernetes":
+        iconClass = "pficon pficon-kubernetes";
+        break;
+      case "atomic":
+        iconClass = "pficon product-vendor-atomic";
```

Comment:
hmm, I wonder why do you need to call it pficon here, it doesn't come from pf, it comes from miq and the font family is different